tests/requirements.txt

```diff
@@ -6,12 +6,12 @@ charset-normalizer==2.0.7
 click==8.0.3
 colorama==0.4.4
 decorator==5.1.0
-frictionless==5.13.1
```

Comment:
We could update this to 5.17.0 so we always test the latest version (or 5.13.1 to indicate that's where we started on).

Reply:
I would test the latest version.

Reply:
Ok, sounds good to me, so updating to >= 5.17.0 then?